oojs/The-keyword-this.md

```diff
@@ -4,7 +4,7 @@ In most classical object oriented languages, there is a way to refer to the curr
 
 Now _this_ maybe the most misunderstood aspect of the language. One way to understand it better is to think of it as the parameter _this_, as it behaves almost exactly as a parameter with two exceptions.
 
-Firstly you, the developer, cannot pick its name - it will always be called `this`.
+Firstly you, the developer, cannot pick its name - it will always be the parameter `this`.
```

Comment:
I have changed the references in the first half, but I think the second half should still use backticks as there it is referencing the code examples.